templates/amp/entry.php

```diff
@@ -0,0 +1,70 @@
+<?php
+	$id = $this->get( 'id' );
+	$entry_time = $this->get( 'time' );
+	$content = $this->get( 'content' );
+	$social = $this->get( 'social' );
+	$single = $this->get( 'single' );
+	$share_link = $this->get( 'share_link' );
+	$update_time = $this->get( 'update_time' );
+	$share_link_amp = $this->get( 'share_link_amp' );
+?>
+
+<div class="liveblog-entry" id="post<?php echo esc_attr( $update_time ); ?>"
+	data-sort-time="<?php echo esc_attr( $entry_time ); ?>">
+
+	<aside class="liveblog-entry-aside">
+		<a class="liveblog-meta-time" href="#" target="_blank">
+			<span><?php echo esc_html( $this->get( 'time_ago' ) ); ?></span>
+			<span><?php echo esc_html( $this->get( 'date' ) ); ?> </span>
+		</a>
+	</aside>
+
+	<div class="liveblog-entry-main">
+		<header class="liveblog-meta-authors">
+
+			<?php if (is_array( $this->get( 'authors' ))): ?>
+
+				<?php foreach ( $this->get( 'authors' ) as $author ) : ?>
+
+					<?php
+					$this->load_part(
+						'author', array(
+							'author' => $author,
+						)
+					);
+					?>
+
+				<?php endforeach ?>
+
+
+				<?php else : ?>
+
+				<?php
+				$this->load_part(
+					'author', array(
+						'author' => $authors,
+					)
+				);
+				?>
+
+			<?php endif; ?>
+		</header>
+
+		<div class="liveblog-entry-content">
+			<?php echo $this->get( 'content' ); ?>
+
+			<?php if ( $single ) : ?>
+				<a href="<?php echo esc_url( $share_link ); ?>">View in feed</a>
+			<?php endif; ?>
+
+			<?php if ( count( $social ) > 1 ) : ?>
```

Comment:
Because the filter to turn off which social platforms can take an empty array.

```php
add_filter( 'liveblog_amp_social_share_platforms', function( $social_array ) {
	return array();
} );
```
Or they may just pass one
```php
add_filter( 'liveblog_amp_social_share_platforms', function( $social_array ) {
	return array(  'twitter' );
} );
```
Or just turn one off.
```php
add_filter( 'liveblog_amp_social_share_platforms', function( $social_array ) {
	unset( $social_array['gplus'] );  
	return $social_array
} );
```

Reply:
D'oh of course